src/Bounds.cpp

```diff
@@ -339,8 +339,10 @@ class Bounds : public IRVisitor {
 
         if (a.is_single_point(op->a) && b.is_single_point(op->b)) {
             interval = Interval::single_point(op);
+            return;
```

Comment:
Worth adding "The bounds of a multiply is just the multiply when there's only a single point involved. No need to check for overflow." as comment here?